src/main/resources/openshift/templates/rhamt-template.json

```diff
@@ -654,6 +654,12 @@
                                 "name": "${APPLICATION_NAME}",
                                 "image": "${APPLICATION_NAME}",
                                 "imagePullPolicy": "Always",
+                                "resources": {
+                                    "requests": {
+                                        "cpu": "2",
+                                        "memory": "4Gi"
```

Comment:
Oh, good catch. We should reduce this to fit into minishift. I wonder if it can be parameterized? I will try that.